Sources/HipMobileUI/Pages/LoadingPage.xaml

```diff
@@ -33,32 +33,33 @@
                         <ColumnDefinition Width="*" />
                     </Grid.ColumnDefinitions>
                     <Grid.RowDefinitions>
-                        <RowDefinition Height="0.4*" />
-                        <RowDefinition Height="0.5*" />
+                        <RowDefinition Height="0.3*" />
+                        <RowDefinition Height="0.6*" />
                         <RowDefinition Height="0.1*" />
                     </Grid.RowDefinitions>
-                    <StackLayout Grid.Row="0" Orientation="Vertical" Spacing="20" HorizontalOptions="CenterAndExpand"
+                    
+                    <StackLayout Grid.Row="0" Orientation="Vertical" Spacing="0" HorizontalOptions="CenterAndExpand" Padding="20" Margin="15"
                                  VerticalOptions="CenterAndExpand">
-                        <Label Text="{Binding Text}" HorizontalTextAlignment="Center" FontSize="20"
-                               FontAttributes="Bold" Style="{DynamicResource Ssp-Bold}" />
-                        <Label Text="{Binding Subtext}" HorizontalTextAlignment="Center" FontSize="16" Style="{DynamicResource Ssp-Regular}"/>
-                        <ProgressBar Progress="{Binding LoadingProgress}" Margin="20,0">
+                        <Label Text="{Binding Text}" HorizontalTextAlignment="Center" FontSize ="20"
+                               FontAttributes="Bold" Style="{DynamicResource Ssp-Bold}"/>
+                        <Label Text="{Binding Subtext}" HorizontalTextAlignment="Center" FontSize="16" Style="{DynamicResource Ssp-Regular}" Margin="10"/>
+                        <ProgressBar Progress="{Binding LoadingProgress}">
                             <ProgressBar.Effects>
                                 <effects:ProgressBarColorEffect Color="{StaticResource PrimaryDarkColor}" />
                             </ProgressBar.Effects>
                         </ProgressBar>
-                        <Label Text="{Binding LoadingProgress, StringFormat='{0:p0}'}" HorizontalTextAlignment="Center"
-                               FontSize="16" Style="{DynamicResource Ssp-Regular}" />
+                        <Label Text="{Binding LoadingProgress, StringFormat='{0:p0}'}" HorizontalTextAlignment="Center" Margin="5"
+                               FontSize="Default" Style="{DynamicResource Ssp-Regular}" />
```

Comment:
If specified as `Default`, I guess removing the attribute altogether (i.e. remove the whole `FontSize=Default`) wouldn't make a difference?

Reply:
removing the "FontSize=Default" makes the font of loading numeric value a bit lighter, changing it to "FontSize=16"